tools/helios-build/src/main.rs

```diff
@@ -2610,14 +2634,14 @@ fn main() -> Result<()> {
     });
     handlers.push(CommandInfo {
         name: "experiment-image".into(),
-        desc: "experimental image construction for Gimlets".into(),
+        desc: "experimental image construction for Compute Sleds".into(),
```

Comment:
This is just a compatibility alias, so it should have the same description.

```suggestion
        desc: "image construction for Compute Sleds".into(),
```